VideoWeb/VideoWeb.UnitTests/Mappings/CourtRoomsAccountResponseMapperTest.cs

```diff
@@ -11,23 +11,22 @@ public class CourtRoomsAccountResponseMapperTest : BaseMockerSutTestSetup<CourtR
         [Test]
         public void Should_map_user_response_to_court_rooms_account()
         {
-            var accounts = UserResponseBuilder.BuildData();
-            var userNames = new List<string> { "Manual01", "Manual03" };
-            var result = _sut.Map(accounts, userNames);
+            var conferences = ConferenceResponseBuilder.BuildData();
+            var result = _sut.Map(conferences);
 
             result.Should().NotBeNull();
             result.Count.Should().Be(2);
             result[0].CourtRooms.Count.Should().Be(3);
             result[1].CourtRooms.Count.Should().Be(1);
 
-            result[0].Venue.Should().Be("Manual01");
-            result[1].Venue.Should().Be("Manual03");
+            result[0].Venue.Should().Be("FirstName1");
```

Comment:
Can you maybe change like "FirstName1" to "Venue1" and "LastName1" to "Room1". Sorry to be picky but it makes the test harder to read.

Reply:
I changed the venues and courtnames to firstName and lastNames in

Reply:
The judge name isn't used any more is it? The variable name Venue and CourtRooms was correct. I was talking about the string comparisons. Can you change the variable names back, they aren't descriptive now. Sorry if I wasn't more clear.

Reply:
The Judge name isn't used in admin/venue-list but still used in admin/hearing-list